### PR TITLE
Upgrade reservation

### PR DESCRIPTION
**Pull request by facuromero1**
this is a upgrade of reservation in which case 3 is ended by adding the purpose of creating the reservation, It also adds the functionality to see the reservations of a specific user.

### CODE REVIEW

File: src/app/Main.java [hunk 1]
```diff
@@ -109,15 +119,14 @@ private static User getUser(Scanner sn) {
 
     private static Book getBook(Scanner sn) {
```
Comment (jlopezscala) [inline src/app/Main.java]: Can you rename this to `getNewBook()` or `createNewBook()` ?

File: src/app/Main.java [hunk 2]
```diff
@@ -126,15 +135,39 @@ public static User userExists(String name, String lastname, ArrayList<User> user
...
+            if (reserva.getUser().getName().equals(name) && reserva.getUser().getLastname().equals(lastname)) {
+                userReservation.add(reserva);
+            }
+        }
+        return userReservation;
+
+    }
+
+
+    public static Reserva reservationExist(Book book, User user, ArrayList<Reserva> reservas) {
```
Comment (jlopezscala) [inline src/app/Main.java]: Where is this being used?

File: src/app/Main.java (hunk 1 above)
Comment (facuromero1) [reply src/app/Main.java]: Okey, no problem.

File: src/app/Main.java (hunk 2 above)
Comment (facuromero1) [reply src/app/Main.java]: This is trash code, I must remove it before merge to master.

**Review by jlopezscala — APPROVED** [review]

**Pull request merged**